databook/physiology/ephys/visual-behavior/VB-Neuropixels.md

```diff
@@ -1,12 +1,12 @@
 # Visual Behavior Neuropixels
 
-The Visual Behavior Neuropixels dataset consists of electrophysiological recordings of neural activity in mice performing a visual change detection task. Each experiment includes data from up to six Neuropixels probes recording simultaneously in cortex, hippocampus, thalamus and midbrain, while mice were exposed to different sensory and behavioral contexts, including familiar and novel stimuli, as well as active and passive stimulus blocks. Data for each experiment is packaged in Neurodata Without Borders (NWB) files that can be accessed via AllenSDK.
+The Visual Behavior Neuropixels dataset consists of electrophysiological recordings of neural activity in mice performing a visual change detection task. Each experiment includes data from up to six Neuropixels probes recording simultaneously in cortex, hippocampus, thalamus and midbrain, while mice were exposed to different sensory and behavioral contexts, including familiar and novel stimuli, as well as active and passive stimulus blocks. Data for each experiment is packaged in Neurodata Without Borders (NWB) files which can be accessed via AllenSDK.
```

Comment:
this should be that

Reply:
I believe it's "which" if all NWB files can be accessed via AllenSDK and "that" if only some NWB files can be accessed in that manner?

Reply:
not all NWB files can be access via SDK.